e2e/browser/src/components/header/index.jsx

```diff
@@ -77,7 +77,7 @@ export default function Header() {
         {!sessionRequestInProgress && !session.info.isLoggedIn && (
           <LoginButton
             authOptions={{ clientName: CONFIG.demoTitle }}
-            oidcIssuer="https://broker.pod.inrupt.com"
+            oidcIssuer="https://openid.dev-next.inrupt.com"
```

Comment:
```suggestion
            oidcIssuer="https://login.inrupt.com"
```

Maybe use the next prod which should be more stable?

Reply:
Currently it appears that editing one's WebID with https://id.inrupt.com/ doesn't work, so I'll keep using dev-next and update this when it is fixed.